build-logic-commons/basics/src/main/kotlin/gradlebuild/basics/kotlindsl/kotlin-compiler-configuration.kt

```diff
@@ -54,8 +54,8 @@ fun CompilerConfiguration.configureKotlinCompilerForGradleBuild() {
     put(
         CommonConfigurationKeys.LANGUAGE_VERSION_SETTINGS,
         LanguageVersionSettingsImpl(
-            languageVersion = LanguageVersion.KOTLIN_1_8,
-            apiVersion = ApiVersion.KOTLIN_1_8,
+            languageVersion = LanguageVersion.KOTLIN_2_1,
```

Comment:
❓ `KOTLIN_2_2`?

Reply:
Not yet, we need to update the wrapper first ;)